dockerfiles/csharp/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.10-alpine
+WORKDIR /go/src/github.com/sourcegraph/lsp-adapter
+COPY . .
+RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-adapter
+
+FROM ubuntu:16.04
+RUN apt-get update && apt-get install -y gnupg
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+RUN apt-get update && apt-get install -y apt-transport-https
+RUN echo "deb https://download.mono-project.com/repo/ubuntu stable-xenial main" | tee /etc/apt/sources.list.d/mono-official-stable.list
+RUN apt-get update && apt-get install -y mono-complete curl
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update && apt-get install -y nodejs
```

Comment:
Wouldn't it be possible to first add all the list entries, then run a single `apt-get update` and `apt-get install`?

Reply:
Not quite, because you have to install `curl` to set up the `nodejs` installation.